sdc/tests/test_rolling.py

```diff
@@ -545,6 +545,29 @@ def test_impl(series, window, min_periods):
                         jit_result = hpat_func(series, window, min_periods)
                         pd.testing.assert_series_equal(ref_result, jit_result)
 
+    @skip_sdc_jit('Series.rolling.sum() unsupported Series index')
+    def test_series_rolling_sum(self):
+        def test_impl(series, window, min_periods):
+            return series.rolling(window, min_periods).sum()
+
+        hpat_func = self.jit(test_impl)
+
+        all_data = [
+            list(range(10)), [1., -1., 0., 0.1, -0.1],
+            [1., np.inf, np.inf, -1., 0., np.inf, np.NINF, np.NINF],
+            [np.nan, np.inf, np.inf, np.nan, np.nan, np.nan, np.NINF, np.NZERO]
+        ]
+        indices = [list(range(len(data)))[::-1] for data in all_data]
+        for data, index in zip(all_data, indices):
+            series = pd.Series(data, index, name='A')
+            for window in range(0, len(series) + 3, 2):
+                for min_periods in range(0, window + 1, 2):
+                    with self.subTest(series=series, window=window,
+                                      min_periods=min_periods):
+                        ref_result = test_impl(series, window, min_periods)
+                        jit_result = hpat_func(series, window, min_periods)
+                        pd.testing.assert_series_equal(ref_result, jit_result)
```

Comment:
I think we should keep jitted result first and result_ref second for consistency to other test files.

Reply:
I agree with that, let me make such change.